README.md

```diff
@@ -54,11 +54,6 @@
 - **[query and update animals table]**
 - **[query multiple tables]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
 
```

Comment:
- [ ] Let's remove the Server `Express.js` from the Tech Stack section since we are not use it here.

![image](https://user-images.githubusercontent.com/79058364/235927358-74378121-3b78-4ba9-b424-159e3dd773ec.png)